src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespacebefore/Example3.java

```diff
@@ -0,0 +1,21 @@
+/*
+NoWhitespaceBefore
+tokens = METHOD_REF, DOT
```

Comment:
Small remark. Config of 3 and 4 are messed up in docs. Not sure we are allowed to have two `tokens` fields
```xml
<module name="NoWhitespaceBefore">
  <property name="tokens" value="METHOD_REF"/>
  <property name="tokens" value="DOT"/>
  <property name="allowLineBreaks" value="true"/>
</module>
```

Reply:
Two token lines is hack in xml config to avoid long lines, during parsing it concatenate all tokens.
You did right conversion.